pkg/base1/cockpit.js

```diff
@@ -2977,7 +3013,7 @@ function full_scope(cockpit, $, po) {
             return low;
         }
 
-        self.load = function load(beg, end) {
+        self.load = function load(beg, end, for_walking) {
```

Comment:
It seems like the arguments here are mutually exclusive. Either beg/end is relevant or for_walking is relevant? Do you think we should just "overload" the function ... so that when no arguments are specified, we load with "for_walking" behavior?